react-router-hono-fullstack-template/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "react-router-hono-fullstack-template",
```

Comment:
@kristianfreeman can you add a description?

Reply:
Added. thanks!